kythe/extractors/extractors.bzl

```diff
@@ -4,7 +4,7 @@ The extractor_action macro plumbs together a Kythe extractor to an extra_action
 with an associated action_listener.
 """
 
-def extractor_action(name, extractor, args, mnemonics, output, needs_output = False, data = [], tags = []):
+def extractor_action(name, extractor, args, mnemonics, output, needs_output = False, data = [], tags = [], env = []):
```

Comment:
`env = {}`

Reply:
Should be fixed now. That's what I get for copy/pasting from a test file